spec/lib/msf/ui/console/command_dispatcher/db/klist_spec.rb

```diff
@@ -23,20 +23,24 @@ def kerberos_ticket_storage
   describe '#cmd_klist' do
     before(:each) do
       kerberos_ticket_storage.delete_tickets
+      ActiveRecord::Base.connection.reset_pk_sequence!('loots')
```

Comment:
I don't like this, I added it so I would have determinsitic IDs for matching the tables below and for tests requiring passing specific ids into the command